Automattic/wp-calypso
# WooCommerce: Update action-type names

Another big cleanup PR. This one updates the action names to match the calypso pattern of suffixing the verb, which allows us to properly alphabetize the list while still grouping related actions. All actions keep the prefix `WOOCOMMERCE`.

The APIPLAN actions were left alone, the rename being done in #14602 is ✅ 

See [the devdocs](https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#actions) for more info on actions in Calypso.

To test:

- Make sure the tests still run
- Check that any section you've hooked up to state still loads
